src/details/ArborX_DetailsTreeConstruction.hpp

```diff
@@ -448,8 +443,8 @@ class CalculateInternalNodesBoundingVolumesFunctor
       // FIXME: accessing Node::bounding_box is not ideal but I was
       // reluctant to pass the bounding volume hierarchy to
       // generateHierarchy()
-      node->bounding_box = (node->children.first)->bounding_box;
-      expand(node->bounding_box, (node->children.second)->bounding_box);
+      node->bounding_box = (_root + node->children.first)->bounding_box;
+      expand(node->bounding_box, (_root + node->children.second)->bounding_box);
```

Comment:
Fix

Reply:
You want to avoid pointer arithmetic here?